src/console_title.go

```diff
@@ -52,12 +45,23 @@ func (t *consoleTitle) getConsoleTitle() string {
 }
 
 func (t *consoleTitle) getTemplateText() string {
-	context := &TitleTemplateContext{
-		Root:   t.env.isRunningAsRoot(),
-		Path:   t.getPwd(),
-		Folder: base(t.getPwd(), t.env),
-		Shell:  t.env.getShellName(),
+	context := make(map[string]interface{})
+
+	context["Root"] = t.env.isRunningAsRoot()
```

Comment:
Check if custom variables should override the environment ones? or if a prefix should be added to environment variables(env_path vs path)?

Reply:
I went with ignoring conflicting environment variables (line 60)

Reply:
sorry, overlooked the continue, my bad.